content/docs/how-we-work/drupal/coding.md

```diff
@@ -27,6 +27,8 @@ If you have written a generic implementation, others in the project can use it v
 - Someone needs help in a drupal.org issue queue, and you know the solution - go ahead and submit your comment
 - You developed a generic solution to a problem. Cool! Let's submit it as a module
 
+We are enthusiastic towards [open source contribution](../../open-source-contribution).
```

Comment:
I think we might be better off without the bullets here entirely. Other sections on this page don't have bullets either. I think it will make this line flow nicely. @Yemaneberhan-Lemma, what do you think of rephrasing the bullets into a paragraph?